Task5/index.html

```diff
@@ -0,0 +1,158 @@
+        <link rel="stylesheet" type="text/css" href="css/main/__grid.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/bestseller-label.css">
+        <link rel="stylesheet" type="text/css" href="css/templates/sale-label.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__button.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__catalog-text.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__sort-text.css">
+        <link rel="stylesheet" type="text/css" href="css/main/__show-more-img.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__image.css">
+        <link rel="stylesheet" type="text/css" href="css/main/main.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__name.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__info.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__old-price.css">
+        <link rel="stylesheet" type="text/css" href="css/main/tour/__price.css">
+    </head>
+    <body>
+        <header >
+            <div class="content header__content">
+                <div class="header__menu-logo">
+                    <img src="images/меню.png">
+                </div>
+                <ul class="header__menu col-menu">
+                    <li class="menu__item menu__user-icon selected-item text">Личный кабинет</li>
+                    <li class="menu__item menu__currency-icon selected-item text">Рубль</li>
+                    <li class="menu__item selected-item text">Ru</li>
+                    <li class="menu__item text"> <strong>8 500-61-05</strong> </br>
+                        Круглосуточная поддержка</p>
+                </ul>
+                <div class="header__label">
+                    <img class="header__logo" src="images/logo.png">
+                    <p class="header__headline line1 text">Морские прогулки</p>
+                    <p class="header__headline line2 text">ПО ЧЁРНОМУ МОРЮ</p>
+                    <button class="simple-button header__button">Выбрать круиз</button>
+                </div>
+            </div>           
+            <img class="header__background-image">
+        </header>
+        <main>
+            <div class="content">
+                <div>
+                    <p class="text main__catalog-text">Каталог круизов</p>
+                    <p class="text main__sort-text selected-item">Сортировать круизы по <em>цене</em></p>
+                </div>
+                <div class="grid tours">
+                    <div class="main__tour tour1 bestseller-label">
+                        <img class="tour__image" src="images/tour1-image-lg.png">
+                        <div class="tour__info">                       
+                            <p class="tour__name tour__show-more-img"><strong>Морской круиз в Анапу</strong></p>   
```

Comment:
`<strong>` is a semantic tag, it shouldn't be used just for styling as here. Use CSS classes for it.

Reply:
Done, moved styling to css